src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+public class Truck extends Mashine {

```

Comment:
The class name `Mashine` seems to be a typo. It should be `Machine` according to the task description and checklist.